### PR TITLE
Updated transaltion-review-and-correction File

### PR DESCRIPTION
**Pull request by purnesh**
Added new UI features like character count, maximum number of rows per block and clearer instructions.

*Issue #, if available:*

*Description of changes:* This update will make the template on Github more consistent with the Translate A2I blog post.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: text/translation-review-and-correction.liquid.html [hunk 1]
```diff
@@ -5,46 +5,52 @@
...
     </div>
 
     <table>
       <tr>
-        <th>Original</th>
-        <th>Translation</th>
-        <th style="width: 70px">Rating</th>
+        <th>Sentence</th>
+        <th>Original ({{ task.input.SourceLanguage }})</th>
+        <th>Translation  ({{ task.input.TargetLanguage }})</th>
```
Comment (samuel-henry) [inline text/translation-review-and-correction.liquid.html]: There's an extra space here

File: text/translation-review-and-correction.liquid.html [hunk 2]
```diff
@@ -5,46 +5,52 @@
...
     padding: 5px;
+    min-width: 100%;
+    background-color: white;
   }
 </style>
 
 <crowd-form>
+  <full-instructions header="Translation Quality Check Instructions">
+    <p>The source document has been broken up into sentences and each sentence has been translated using <a href="https://aws.amazon.com/translate/" target="blank" >Amazon Translate</a>. The source text, along with their
+      <strong>machine generated translation</strong> is shown in the table. Please review the translated text and make changes so the read more like something a person would write rather than an automated translation.
```
Comment (samuel-henry) [inline text/translation-review-and-correction.liquid.html]: typo: "so the read more" should be something like "so it reads more"

File: text/translation-review-and-correction.liquid.html (hunk 1 above)
Comment (purnesh) [reply text/translation-review-and-correction.liquid.html]: Removed

File: text/translation-review-and-correction.liquid.html (hunk 2 above)
Comment (purnesh) [reply text/translation-review-and-correction.liquid.html]: Done and added a Strong tag around Submit to emphasize that it's a button.

**Review by samuel-henry — APPROVED** [review]

**Pull request merged**